prediction_market_agent/db/pinecone_handler.py

```diff
@@ -109,15 +112,22 @@ def deduplicate_markets(markets: list[OmenMarket]) -> list[OmenMarket]:
 
         return list(unique_market_titles.values())
 
-    def insert_all_omen_markets_if_not_exists(
-        self, created_after: DatetimeUTC | None = None
+    def update_markets(self) -> None:
```

Comment:
- Can remove this function as it just wraps `insert_open_omen_markets_if_not_exists`
- `insert_all_omen_markets_if_not_exists` has been changed to `insert_open_omen_markets_if_not_exists`, but it's (nearly) the same as the old function (i.e. not getting open markets only). So no need for rename.
- `created_after: DatetimeUTC` has been changed to `start_timestamp: int`. But `OmenSubgraphHandler.get_omen_binary_markets_simple(created_after=...)` takes `DatetimeUTC`. So should change back

Reply:
3 points implemented here https://github.com/gnosis/prediction-market-agent/commit/9889efc925d5ae57bf04694aacb20e56a37e5042